### PR TITLE
fallback i18n

### PR DESCRIPTION
**Pull request by jasl**
I18n will broken when locale isn't `en`.

image

### CODE REVIEW

File: lib/doorkeeper/engine.rb [hunk 1]
```diff
@@ -4,6 +4,11 @@ class Engine < Rails::Engine
       app.config.filter_parameters += %i(client_secret code token)
     end
 
+    initializer "doorkeeper.locales" do |app|
+      app.config.i18n.fallbacks = true
```
Comment (tute) [inline lib/doorkeeper/engine.rb]: This line is not needed, next one should suffice.

**Comment by jasl** [comment]
fixed

**Comment by tute** [comment]
Thanks! Will hold merge until next week, when I have time to investigate why Travis is failing.

Thanks for your other PRs too!

**Comment by jasl** [comment]
Plz review #629 first, I resolve the Travis-CI issue.

**Comment by tute** [comment]
Thank you!

**Comment by tute** [comment]
@jasl I'll give you commit access to doorkeeper org so we can collaborate together. Please don't push to master without code reviews! Thank you for all your great work! :smiley: :+1: :heart:

**Comment by jasl** [comment]
That's my honor

**Comment by tute** [comment]
You might want to enable 2 factor authentication to your GitHub account and associated email, to decrease the odds someone else can impersonate your identity in GitHub.

**Pull request merged**